react/utils/pixelHelper.ts

```diff
@@ -0,0 +1,78 @@
+export function mapCartItemToPixel(item: CartItem): PixelCartItem {
+  return {
+    skuId: item.id,
+    variant: item.skuName,
+    price: item.sellingPrice,
+    name: item.name,
+    quantity: item.quantity,
+    productRefId: item.productRefId,
+    brand: item.additionalInfo ? item.additionalInfo.brandName : '',
+    category: productCategory(item),
+  }
+}
+
+export function mapBuyButtonItemToPixel(item: BuyButtonItem): PixelCartItem {
+  // Change this `/Apparel & Accessories/Clothing/Tops/`
+  // to this `Apparel & Accessories/Clothing/Tops`
+  const category = item.category
+    ? item.category.slice(1).slice(0, -1)
```

Comment:
⛏  why not:
```suggestion
    ? item.category.slice(1, -1)
```

or I'm missing something?